var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -56,13 +64,14 @@ class Scorep(Package):
     depends_on("papi")
 
     def install(self, spec, prefix):
-        configure = Executable( join_path(self.stage.source_path, 'configure') )
+        configure = Executable(join_path(self.stage.source_path, 'configure'))
         with working_dir('spack-build', create=True):
             configure_args = ["--prefix=%s" % prefix,
```

Comment:
For future reference: I like to break lines right after the opening `[`, as this tends to lead reduce the indentation of the whole list, and one doesn't need to split individual list items over multiple lines.